packages/wallet-service/tests/utils/pushnotification.utils.test.ts

```diff
@@ -6,7 +6,8 @@ import { sendMulticastMock, messaging, initFirebaseAdminMock } from '@tests/util
 import { logger } from '@tests/winston.mock';
 import { PushNotificationUtils, PushNotificationError, buildFunctionName, FunctionName } from '@src/utils/pushnotification.utils';
 import * as pushnotificationUtils from '@src/utils/pushnotification.utils';
-import { SendNotificationToDevice, Severity } from '@src/types';
+import { SendNotificationToDevice } from '@src/types';
+import { Severity  } from '@wallet-service/common/src/types';
```

Comment:
style: Lint fix
```suggestion
import { Severity } from '@wallet-service/common/src/types';
```

Reply:
Done in 661c1478522b0b101dbb8f08015e2b466d2f3d2c